labcontrol/db/process.py

```diff
@@ -4032,9 +4032,8 @@ def _get_metagenomics_data_for_prep(self):
                 # query.
                 d['instrument_model'] = inst_mdl
 
-                # note that the correct term is 'Kapa', not 'kappa'.
-                id = d['kappa_hyper_plus_kit_id']
-                d['kappa_hyper_plus_kit_lot'] = reagent[id]['external_lot_id']
+                id = d['kapa_hyper_plus_kit_id']
```

Comment:
I had the impression that `id` is a reserved Python keyword. So I usually use `id_`.

Reply:
To my knowledge, 'id' is not reserved. I know that 'if', 'is', and 'in' are.